modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/parser/java/JavaConfigurationModelParser.java

```diff
@@ -69,6 +71,7 @@ public JavaConfigurationModelParser(JavaExtensionModelParser extensionModelParse
     this.configElement = configElement;
 
     parseStructure();
+    this.minMuleVersion = getMinMuleVersion().orElse(FIRST_MULE_VERSION);
```

Comment:
I would also add this to the Extension parser, in order to check there is no issue with those too.

Reply:
done